src/main/resources/static/_shared.yml

```diff
@@ -1608,6 +1608,8 @@ components:
               type: boolean
             isEsapApplication:
               type: boolean
+            isMhapApplication:
```

Comment:
Do we absolutely need to add this given the new status? Is this a UI requirement to help migrate over?

Reply:
You're right, I've removed this as it would be easier to just use `apType` instead of introducing an intermediate step.